__mocks__/rehype-autolink-headings.js

```diff
@@ -0,0 +1,13 @@
+/* Manual Mocking the rehype-autolink-headings library because our jest and typscript
+ * configuration doesn't use esm modules yet. After the package update,
+ * rehype-autolink-headings is a fully ESM module now and the reason this is being
+ * manual mocked is because it is being mocked as a CommonJS module.
+ * Here is the link: https://jestjs.io/docs/manual-mocks#mocking-node-modules
+ */
+const autolink = () => {
```

Comment:
After updating nextjs, is this essential?

Reply:
Yeah because the way jest mocks these packages is as a commonJS module. RIght now our jest isn't configured to use ES modules